test/util/cart_utils.py

```diff
@@ -40,8 +45,11 @@ def __init__(self):
         """ CartUtils init """
         self.stdout = logging.getLogger('avocado.test.stdout')
         self.progress_log = logging.getLogger("progress")
+        self.module_init = False
```

Comment:
`mpi_module_init` perhaps?

Reply:
technically, any environment module could be loaded but I can change if you prefer

Reply:
Oh, right.  I see what this is flagging now.